FunctionalTableData/CollectionView/FunctionalCollectionData.swift

```diff
@@ -507,7 +489,7 @@ extension FunctionalCollectionData: UICollectionViewDelegate {
 			}
 		}
 	}
-
+	
```

Comment:
Can you fix all of these please?

Reply:
Actually, these appeared because I reformatted the file with tabs locally - if you take a look at the rest of the file, 
all extra lines will have this indentation - this is actually the correct format